tests/TestWorker.py

```diff
@@ -249,10 +249,9 @@ def test_InvalidEncoding(self):
         """Test Worker using Action.InvalidEncodingAction"""
         self.action_test_helper('invalid.encoding', 0, 0, 4096, 2, 3, 2)
 
+    @unittest.skipUnless('nt' == os.name, 'skipping on non-Windows')
```

Comment:
One next step is to include `from __future__ import absolute_import`, but this complicates the imports in tests. Currently, I'm using the [unittest2 test discovery](https://docs.python.org/2/library/unittest.html#test-discovery), but this runs all tests.
Skipping the tests lets me use either the PyCharm test runner or the unittest2 test discovery.